### ISSUE TITLE
There is no prompt after the session expires. You need to refresh manually to return to the login page

### ISSUE THREAD

**Issue by WuJun2016** [rancher/dashboard#6686]
**Setup**
- Rancher version:
- Browser type & version:

**Describe the bug**

There is no prompt after the session expires. You need to refresh manually to return to the login

**To Reproduce**

1. Log in to rancher
2. Wait about ten hours
3. No response when clicking on the page
4. Manually refresh and return to the login page

**Result**

**Expected Result**

Click the page prompt to automatically return to the login page after the session expires

**Screenshots**

**Additional context**

**Detailed Description**

**Context**

**Comment by richard-cox** [comment]
The dashboard should return the user to the log in page when it receives a 401 response.

When you say there's no response when clickign page, has the dashboard frozen? Are http requests being made and failing? What page are you on?

Alex added a change in 2.6.7 that will make a basic request to check the session when a user returns to a tab @aalves08 does this also cover cases where the tab becomes visibly again (after unlocking OS, opens from the tray, etc)?

**Comment by aalves08** [comment]
> The dashboard should return the user to the log in page when it receives a 401 response.
> 
> @WuJun2016 When you say there's no response when clicking the page, has the dashboard frozen? Are http requests being made and failing? What page are you on? Finally what version is this in?
> 
> Alex added a [change](https://github.com/rancher/dashboard/pull/5872) in 2.6.7 that will make a basic request to check the session when a user returns to a tab @aalves08 does this also cover cases where the tab becomes visibly again (after unlocking OS, opens from the tray, etc)?

Yes @richard-cox  for after unlocking OS, opens from the tray, switching tabs. There might be some scenarios which we haven't thought about that need testing, but those were covered. You can check on the `Network` tab under Chrome Dev Tools a request to `v3/users?me=true` on each one of those scenarios. If you think of others and want to check if it's working, just for that request.

**Comment by richard-cox** [comment]
Closing, no additional information.

This may already be fixed already, either by Alex's change or the incoming session idle modal
